fission/src/ui/panels/DebugPanel.tsx

```diff
@@ -85,14 +85,14 @@ const DebugPanel: React.FC<PanelPropsImpl> = ({ panelId }) => {
                     <Button
                         value={"Refresh APS Token"}
                         onClick={async () =>
-                            APS.isSignedIn() && APS.refreshAuthToken((await APS.getAuth())!.refresh_token, true)
+                            (await APS.isSignedIn()) && APS.refreshAuthToken((await APS.getAuth())!.refresh_token, true)
```

Comment:
Nice catch for changing `isSingedIn()` to be a async function.

I do believe there is a potential race condition in this line since `APS.getAuth()` is called twice in this line and state can technically change within those two calls. Should save this state to a variable rather than calling these functions inline.